code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -75,7 +75,7 @@
 
 	icon_state = pick("antitoxin","antitoxfirstaid","antitoxfirstaid2","antitoxfirstaid3")
 
-	new /obj/item/weapon/reagent_containers/syringe/antitoxin(src)
+	new /obj/item/weapon/reagent_containers/syringe/antiviral(src)
```

Comment:
I don't directly object to this but it's an example of how you're touching a lot of stuff here that will impact people. This is going to need to eventually be in the changelog - which I am afraid you will forget since this won't be going to the main server right away.

I've taken a look at the spaceacillin RGB and fortunately it does look fairly different I believe, so at least there's that.

Reply:
yeah, I've updated spaceacillin so it looks pinkish and is easy to differentiate. The changelog will be massive, but I'll be going through every line of the final merge PR to make it like I usually do, so I don't think I'll forget this change, especially since it goes hand in hand with helping the playerbase making peace with viro.